zcash_client_sqlite/src/error.rs

```diff
@@ -78,7 +81,7 @@ impl fmt::Display for SqliteClientError {
                 write!(f, "The note ID associated with an inserted witness must correspond to a received note."),
             SqliteClientError::RequestedRewindInvalid(h, r) =>
                 write!(f, "A rewind must be either of less than {} blocks, or at least back to block {} for your wallet; the requested height was {}.", PRUNING_HEIGHT, h, r),
-            SqliteClientError::Bech32(e) => write!(f, "{}", e),
+            SqliteClientError::KeyError(e) => write!(f, "{}", e),
```

Comment:
We should ensure we still tell the user what kind of data failed to be parsed:
```suggestion
            SqliteClientError::KeyError(kind, e) => write!(f, "Not a valid {}: {}", kind, e),
```